.golangci.yml

```diff
@@ -15,6 +15,12 @@ issues:
   # Set to 0 to disable.
   # Default: 50
   max-issues-per-linter: 50
+  exclude-rules:
+    # Exclude package name contains '-' issue because we have at least one package with
+    # it on its name.
+    - linters:
+        - stylecheck
+      text: "ST1003:"
```

Comment:
```suggestion
      text: "ST1003:" # Poorly chosen identifier
```

Reply:
I had it at the top, but I agree the yaml structure didn't help:
```
    # Exclude package name contains '-' issue because we have at least one package with
    # it on its name.
```

What about having it like that:
```yaml
    # Exclude package name contains '-' issue because we have at least one poorly chosen
    # package with'-' it on its name.
    - text: "ST1003:" 
      linters:
        - stylecheck
```

Reply:
I missed that, I thought that comment was not related to the particular exception but now I see it's a list and the comment is on top of the item.

Reply:
The original yaml snipped I copied from is pretty confusing, but after taking some time to decipher it, I came up with something I hope is more readable.